dictionary.txt

```diff
@@ -275,13 +290,15 @@ MIMcAllister
 miscfxn
 misspecification
 MLE
+mle
 MLEnew
 ModelDimensions
 modelname
 mortalities
 Mohn
-MEY
```

Comment:
I don't think I fully understand what is listed in this file but I would not expect MEY to be deleted since this is an abbreviation in the document.

Reply:
The dictionary.txt file is just the list of words for the github action spell check to ignore. I'm guessing that the action updated the file to remove words that no longer occur in the document. Users of the manual will never be impacted by the contents of this file.

The new file ss3_glossaries.tex is where the glossary words are stored.